Packages/io.chainsafe.web3-unity/Runtime/Scripts/EVM/Token/Methods.cs

```diff
@@ -1,6 +1,6 @@
 namespace Scripts.EVM.Token
 {
-    public static class CommonMethod
+    public static class EthMethod
```

Comment:
Consider, `TokenMethod`

Reply:
Eth method was what we were using before in the other scripts that weren't common method, I merged them. I honestly prefer eth method though. Don't think it matters too much.